distribution/ddf-common/src/main/resources/bin/setenv-windows-wrapper.conf

```diff
@@ -0,0 +1,58 @@
+# Enables restart of JVM when it exits with 10. 10 is normally used by the karaf script to
```

Comment:
ok, you got me 😄 
I had to provide this file since I was changing the documentation on how to install the wrapper. That being said, the file doesn't define the error handling support yet; it is commented out and left for the next ticket.